pkgdb2/__init__.py

```diff
@@ -83,6 +83,34 @@
 SESSION = pkgdblib.create_session(APP.config['DB_URL'])
 
 
+def _monkey_patch_jsonify_jsonp():
+    """ Monkey patch Flask's "jsonify" to also handle JSONP
+
+    This makes it so that all API endpoints that return JSON data can now also
+    return JSONP data.  This is used specifically by some of our apps that want
+    to make ajax calls from one app to pkgdb for more information.
+    """
+
+    original_jsonify = flask.jsonify
+
+    def _jsonify_with_jsonp(*args, **kwargs):
+        response = original_jsonify(*args, **kwargs)
+
+        callback = flask.request.args.get('callback', None)
+
+        if callback:
```

Comment:
Could we check that the request is a GET and never a POST? Since the API is r/w I would prefer that we do not support jsonp on POST queries.